lib/src/spec/message.dart

```diff
@@ -2,16 +2,40 @@ import 'enums.dart';
 import 'rest/channels.dart';
 
 class Message {
-  Message.fromEncoded(Map jsonObject, ChannelOptions channelOptions);
-  Message.fromEncodedArray(List<Map> jsonArray, ChannelOptions channelOptions);
+
```

Comment:
Are there any comments in similar classes in implementations for other languages that are more elaborate/exact?
If there are, it might be worth considering to copy them, to perhaps have better comments
and comments that are at least somewhat consistent with the other implementations.

"The timestamp for this message" is a quite redundant comment.

Reply:
These comments are actually copied from java implementation

Reply:
that's sad🥴

Reply:
Ok, I leave it at that for now. 
Perhaps when I understand the API better I can come up with a better comment.